evennia/web/website/forms.py

```diff
@@ -0,0 +1,142 @@
+from django import forms
+from django.conf import settings
+from django.contrib.auth.forms import UserCreationForm, UsernameField
+from evennia.utils import class_from_module
+from random import choice, randint
+
+class AccountCreationForm(UserCreationForm):
+    
+    class Meta:
+        model = class_from_module(settings.BASE_ACCOUNT_TYPECLASS)
+        fields = ("username", "email")
+        field_classes = {'username': UsernameField}
+    
+    email = forms.EmailField(help_text="A valid email address. Optional; used for password resets.", required=False)
+    
+class CharacterCreationForm(forms.Form):
+    name = forms.CharField(help_text="The name of your intended character.")
+    age = forms.IntegerField(min_value=3, max_value=99, help_text="How old your character should be once spawned.")
+    description = forms.CharField(widget=forms.Textarea(attrs={'rows': 3}), max_length=2048, min_length=160, required=False)
+    
+    @classmethod
+    def assign_attributes(cls, attribute_list, points, min_points, max_points):
```

Comment:
This appears to be game specific and does not belong in core as far as I can tell?

Reply:
Inclusion on this form is debatable; it's more of a general utility function. It might be better suited to inclusion in utils or just a contrib. Most games involving rolling initial stats and distributing points end up having to come up with a similar algorithm for this anyway.